lib/mercury/received_message.rb

```diff
@@ -38,14 +38,12 @@ def nack
     private
 
     def is_ackable
-      @work_queue_name != nil
+      !@work_queue_name.nil?
     end
 
     def performing_action(action)
-      is_ackable or raise "This message is not #{action}able"
-      if @action_taken
-        raise "This message was already #{@action_taken}ed"
-      end
+      is_ackable || raise("This message is not #{action}able")
+      raise "This message was already #{@action_taken}ed" if @action_taken
```

Comment:
@crgee Yeah, if there's a bunch of `raise ... if ...` lines at the _beginning_ of a method, that's fine because you can just ignore the entire chunk (unless you're specifically thinking about error handling).